frontend/src/metabase/css/components/tippy.css

```diff
@@ -0,0 +1,3 @@
+.tippy-content {
+  white-space: pre-wrap;
+}
```

Comment:
There are actually lots of references to the current TippyPopover style in https://github.com/metabase/metabase/blob/master/frontend/src/metabase/components/Popover/Popover.css

And that style is imported in
https://github.com/metabase/metabase/blob/60502ec8acc8de8eecf07a37822437c9f807d95a/frontend/src/metabase/components/Popover/Popover.jsx#L10

Maybe we should move `Popover.css` to `metabase/css/` and move this style to that file. What do you think @daltojohnso

Reply:
Thank you for pointing this out, @WiNloSt.

As far as I see, Tooltip is not as close to Popover as we may think at first.

Popover uses the Tippy library, but Tooltip doesn't use Popover.

That said, maybe there is a much better way of organizing these files, but possibly not this style being moved into Popover styles.